tests/phpunit/tests/block-supports/elements.php

```diff
@@ -3,171 +3,306 @@
  * @group block-supports
  *
  * @covers ::wp_render_elements_support
+ * @covers ::wp_render_elements_support_styles
  */
 class Tests_Block_Supports_Elements extends WP_UnitTestCase {
 	/**
-	 * Given a string containing a class prefixed by "wp-elements-" followed by a unique id,
-	 * this function returns a string where the id is one instead of being unique.
-	 *
-	 * @param string $block_content String containing unique id classes.
-	 * @return string String where the unique id classes were replaced with "wp-elements-1".
+	 * @var string|null
 	 */
-	private static function make_unique_id_one( $block_content ) {
-		return preg_replace( '/wp-elements-[a-zA-Z0-9]+/', 'wp-elements-1', $block_content );
+	private $test_block_name;
+
+	public function set_up() {
+		parent::set_up();
+		$this->test_block_name = null;
+	}
```

Comment:
This fixture method shouldn't be needed as `$this->test_block_name` is already reset in `::tear_down()`.

Reply:
Good catch, I've cleaned that up in https://github.com/WordPress/wordpress-develop/pull/5418/commits/a7dbbb7d5c08775aaab81a414d6977137bfef605.